src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/exceptionhandlers/ErrorNotificationExceptionHandler.java

```diff
@@ -1,38 +1,43 @@
 package uk.gov.hmcts.reform.bulkscanprocessor.exceptionhandlers;
 
-import com.microsoft.azure.servicebus.IMessage;
 import uk.gov.hmcts.reform.bulkscanprocessor.client.ErrorNotificationException;
-import uk.gov.hmcts.reform.bulkscanprocessor.model.ErrorNotificationMessageWrapper;
+import uk.gov.hmcts.reform.bulkscanprocessor.services.servicebus.MessageAutoCompletor;
+
+import java.util.UUID;
+import java.util.concurrent.CompletableFuture;
 
 public class ErrorNotificationExceptionHandler {
```

Comment:
Something I didn't think about earlier - it would be good to add some logging, so that we can track the history of the message.

Reply:
Surely. Must've gone between multiple branches I have locally. ..or was not there at all. Gimme couple minutes

Reply:
it may also be useful to log what we decide to do with the message itself (not just what the problem was).
Perhaps in the autocompletor class.